install.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/bash
+
+#@author: dtechshield 
+
+#This assumes all is installed under /opt/HardwareCheckout
+
+GITREPO=https://github.com/linted/HardwareCheckout.git
+APP_PATH=/opt/HardwareCheckout
+DBNAME=chvappdb
+DBUNAME=chvapp
+DBPASS=$(head -10 /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | sort -r | head -n 1)
+
+echo "Run this once and you should be set... Make sure to run it as root!"
+
+if [ ! -d "$APP_PATH" ]; then
+
+  while true; do
+    read -p "$APP_PATH does not exist - do you want me to clone the repo? (Y|N)?" yn
```

Comment:
Usually the prompt will have `y/N` where the capital letter is the default if they don't put in a valid option.